librecad: 2.2.0-rc2 -> 2.2.0

austinbutler opened the pull request:
###### Description of changes

https://github.com/LibreCAD/LibreCAD/releases/tag/2.2.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

austinbutler commented:
Result of `nixpkgs-review pr 194280` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>librecad</li>
  </ul>
</details>

LeSuisse reviewed (changes requested):
Hi,

Would you mind updating the PR to the stable 2.2.0 release?

https://github.com/LibreCAD/LibreCAD/blob/2.2.0/CHANGELOG.md

LeSuisse commented on pkgs/applications/misc/librecad/default.nix:
```diff
@@ -15,23 +15,15 @@
   };
 
-  patches = [
-    (fetchpatch {
```
`fetchpatch` can be removed L4.

austinbutler committed:
librecad: 2.2.0-rc2 -> 2.2.0

austinbutler commented:
Thanks @LeSuisse. Made those changes. Also removed a stale substitution and moved some pieces per [nixpkgs-hammer](https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md).

LeSuisse reviewed (approved):
Build and run fine x86_64.


Result of `nixpkgs-review pr 194280` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>librecad</li>
  </ul>
</details>

IzumiRaine commented:
Could this be backported to 22.11?

LeSuisse commented:
Changelog shows no breaking change so yes we probably can.

https://github.com/LibreCAD/LibreCAD/blob/2.2.0/CHANGELOG.md
